test/blackbox-tests/test-cases/dune-cache/basic-copy.t/run.t

```diff
@@ -27,31 +27,53 @@ It's a duck. It quacks. (Yes, the author of this comment didn't get it.)
   > EOF
 
 Test that after the build, the files in the build directory have the hard link
-count of 2, because they are shared with the corresponding cache entries.
+counts greater than 1, because they are shared with the corresponding cache entries.
 
-  $ dune build --config-file=config target
+  $ dune build --config-file=config target1 --debug-cache=shared,workspace-local
   $ dune_cmd stat hardlinks _build/default/source
+  3
+  $ dune_cmd stat hardlinks _build/default/target1
+  3
+  $ dune_cmd stat hardlinks _build/default/target2
   2
-  $ dune_cmd stat hardlinks _build/default/target
-  2
-  $ ls _build/default/beacon
-  _build/default/beacon
+  $ dune_cmd exists _build/default/beacon
+  true
+  $ cat _build/log | grep '_build/default/source\|_build/default/target'
+  # Workspace-local cache miss: _build/default/source: never seen this target before
+  # Shared cache miss [8b39c1a0b45579f8da18f42be8e6aca0] (_build/default/source): not found in cache
+  # Workspace-local cache miss: _build/default/target1: never seen this target before
+  # Shared cache miss [fccfd1af13c64ce19b45e2a76fb8132c] (_build/default/target1): not found in cache
 
 Test that rebuilding works.
```

Comment:
It took me a bit of time to realise that what I should look for bellow is the fact that we see "workspace-local" cache misses, but no shared cache local misses. Might be worth explaining.

Reply:
Indeed, I added some clarifications about the kind of cache misses that we expect to see and why.